airflow/utils/logging.py

```diff
@@ -71,7 +71,7 @@ def read(self, remote_log_location, return_error=False):
             try:
                 s3_key = self.hook.get_key(remote_log_location)
                 if s3_key:
-                    return s3_key.get_contents_as_string().decode()
+                    return s3_key.get_contents_as_string()
```

Comment:
I believe it's because in Python 3 it returns bytes but is handled as a string elsewhere.